### PR TITLE
Feature/#37/vision_transformer_layer

### PR DESCRIPTION
**Pull request by zassou65535**
## 概要

I-JEPAのモデルに必要となるVision Transformer Layerの導入、テストの追加

## Submit前の確認項目

- [x] タイトルは一目でわかるようにし、説明文は PR を簡潔に説明するようにしましたか?
- [x] あなたの PR が、異なる変更を束ねたものではなく、ひとつのことだけを行うものであることを確認しましたか?
- [x] この PR で導入されたすべての変更点をリストアップしましたか?
- [x] PR を`make run`コマンドでローカルでテストしましたか?

### CODE REVIEW

File: ami/models/components/vision_transformer_layer.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+class MLP(nn.Module):
+    """Multi Layer Perceptron."""
+
+    def __init__(
+        self,
+        in_features: int,
+        hidden_features: int,
+        out_features: int,
+        drop: float = 0.0,
```
Comment (Geson-anko) [inline ami/models/components/vision_transformer_layer.py:21]: `drop`は動詞なので`dropout`と表記するのが良いです。
```suggestion
        dropout: float = 0.0,
```

**Review by Geson-anko — APPROVED** [review]
@zassou65535 コメントapproveします

File: ami/models/components/vision_transformer_layer.py (hunk 1 above)
Comment (zassou65535) [reply ami/models/components/vision_transformer_layer.py]: https://github.com/MLShukai/ami-vconf24/pull/40/commits/085b06ed95928eebdfc6fc4572c49fdf1c149981
で関係しそうな箇所を修正
